### ISSUE TITLE
WebUI won't load

### ISSUE THREAD

**Issue by xombiemp** [hadouken/hadouken#185]
I'm trying to get hadouken running for the first time.  I'm using a fresh install of Ubuntu 14.04:
Linux ubuntu-14 3.16.0-45-generic #60~14.04.1-Ubuntu SMP Fri Jul 24 21:16:23 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

All I've done is update fully with apt-get and then run
`sudo dpkg -i hadouken-5.2.0-amd64.deb`

I copied /etc/hadouken/hadouken.json.template to ~/hadouken.json and edited the defaultSavePath and the statePath then ran it with:
`hadouken --config="/home/andrew/hadouken.json"`

I then see this in my terminal and hadouken seems to be running:

> (00000001) 08:49:08.268816: <info> Loaded config file "/home/andrew/hadouken.json"
> (00000002) 08:49:08.274595: <info> Loading webui cache from "/usr/share/hadouken/webui.zip".
> (00000003) 08:49:08.306729: <debug> 105 items in webui cache.
> (00000004) 08:49:08.371666: <info> Found 14 RPC method(s).
> (00000005) 08:49:08.372900: <info> Enabling DHT.
> (00000006) 08:49:08.373000: <info> Enabling UPnP.
> (00000007) 08:49:08.390405: <info> Loaded session settings.
> (00000008) 08:49:08.391361: <info> Loaded session state from '/home/andrew/.session_state'
> (00000009) 08:49:08.396622: <info> Loaded plugin '/usr/share/hadouken/js/plugins/automove.js'
> (0000000a) 08:49:08.398573: <info> Loaded plugin '/usr/share/hadouken/js/plugins/autolabel.js'
> (0000000b) 08:49:08.400679: <info> Loaded plugin '/usr/share/hadouken/js/plugins/pushover.js'
> (0000000c) 08:49:08.404323: <info> Loaded plugin '/usr/share/hadouken/js/plugins/pushbullet.js'
> (0000000d) 08:49:08.407259: <info> Loaded plugin '/usr/share/hadouken/js/plugins/autoadd.js'
> (0000000e) 08:49:08.410539: <info> Loaded plugin '/usr/share/hadouken/js/plugins/launcher.js'

But when I browse to http://10.2.100.185:7070/gui/index.html , which is the IP of the VM I have this running in, it just spins forever and never loads anything.

**Comment by vktr** [comment]
@jkaberg did you experience the same issue? Did it work for you in 5.1?

**Comment by simonbcn** [comment]
Same problem on Ubuntu 14.04.3 64 bits with Hadouken 5.2:

```
hadouken 
(00000001) 15:20:02.111045: <info> Loaded config file "/home/simon/hadouken.json"
(00000002) 15:20:02.114406: <info> Loading webui cache from "/usr/share/hadouken/webui.zip".
(00000003) 15:20:02.146139: <debug> 105 items in webui cache.
(00000004) 15:20:02.189655: <info> Found 14 RPC method(s).
(00000005) 15:20:02.190191: <info> Enabling DHT.
(00000006) 15:20:02.190610: <info> Loaded session settings.
(00000007) 15:20:02.190909: <info> Loaded session state from '[STATE_PATH]/.session_state'
(00000008) 15:20:02.194296: <info> Loaded plugin '/usr/share/hadouken/js/plugins/automove.js'
(00000009) 15:20:02.195963: <info> Loaded plugin '/usr/share/hadouken/js/plugins/autolabel.js'
(0000000a) 15:20:02.198017: <info> Loaded plugin '/usr/share/hadouken/js/plugins/pushover.js'
(0000000b) 15:20:02.199542: <info> Loaded plugin '/usr/share/hadouken/js/plugins/launcher.js'
(0000000c) 15:20:02.201227: <info> Loaded plugin '/usr/share/hadouken/js/plugins/autoadd.js'
(0000000d) 15:20:02.202674: <info> Loaded plugin '/usr/share/hadouken/js/plugins/pushbullet.js'
```

But when I try to load `http://localhost:7070/gui/index.html` on Firefox 40 and Google Chrome, it does load anything.

**Comment by vktr** [comment]
Thanks, I'm adding this to the v5.3 milestone. Hopefully I'll have a fix within a few days.

**Comment by jkaberg** [comment]
@vktr Never tried 5.1 on that machine

**Comment by vktr** [comment]
I can confirm the webui doesn't load on 5.2 (on Linux, Windows is unaffected), so I wouldn't recommend updating. I do know it worked in 5.1 so something broke between versions.

Sorry for this.

**Comment by frogcrush** [comment]
I have this problem as well, how long do you think it will be until 5.3 is released?

**Comment by vktr** [comment]
@frogcrush Yeah, sorry for this. I'll try to release v5.3 this week or early next week!

**Comment by Dilergore** [comment]
Hi, any news on the update? I can't wait to try Hadouken on my VM but it's on 14.04.3 :-( Thx

**Comment by vktr** [comment]
Sorry for the delay - I've been busy with real life.

I have identified the issue and have a "working" fix in this PR, https://github.com/hadouken/hadouken/pull/199. However, there's a bit more work until it's done, and the RSS functions in libtorrent are deprecated as well, so have to find another solution for that.

You should be able to build 5.1 on 14.04.3 if you get GCC4.9 and follow the old build instructions here, http://docs.hdkn.net/en/v5.1.0/building/ubuntu.html.

Again, sorry for the delay.

**Comment by Dilergore** [comment]
Thanks,

I'll compile 5.1 afternoon :-)

**Comment by danpowell88** [comment]
I dont suppose theres any fix for this going to happen anymore?
